code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -350,15 +325,21 @@
 	S.key = T.key
 	if(U)
 		S.faction |= "\ref[U]" //Add the master as a faction, allowing inter-mob cooperation
-	if(U && iscultist(U))
+		if(!iscultist(U))
+			S.mind.store_memory("<b>Serve [U.real_name], your creator.</b>")
+	if(iscultist(U))

```

Comment:
I believe this should be indented one more to the right now that you don't check for `U` nul-ness

Reply:
Right, thanks for catching, I'm half assing these review changes which I shouldn't be.